_notices/rsn0030.md

```diff
@@ -7,37 +7,33 @@ notice_type: rsn
 # Update meta-data for notice
 notice_id: 30 # should match notice number
 notice_pin: true # set to true to pin to notice page
-title: "Docker Image Changes in Release ~~v23.06~~ v23.08"
+title: "Docker Image Changes in Release v23.08"
 notice_author: RAPIDS TPM
-notice_status: In Progress
-notice_status_color: yellow
+notice_status: Completed
+notice_status_color: green
 # 'notice_status' and 'notice_status_color' combinations:
 #   "Proposal" - "blue"
 #   "Completed" - "green"
 #   "Review" - "purple"
 #   "In Progress" - "yellow"
 #   "Closed" - "red"
 notice_topic: Platform Support Change
-notice_rapids_version: "v23.06"
+notice_rapids_version: "v23.08"
 notice_created: 2023-04-20
 # 'notice_updated' should match 'notice_created' until an update is made
-notice_updated: 2023-05-24
+notice_updated: 2023-08-17
 ---
 
 ## Overview
 
-**THIS CHANGE IS BEING POSTPONED UNTIL RELEASE v23.08**
-
-RAPIDS is making a number of changes to its end-user Docker images to improve functionality, performance, and maintainability. These changes are targeted to be introduced in the RAPIDS 23.08 release.
+RAPIDS is making a number of changes to its end-user Docker images to improve functionality, performance, and maintainability. These changes are targeted to be completed in the RAPIDS 23.08 release.

```

Comment:
Do we want to put this in past tense, to indicate that it has happened (rather than targeted for the future)?
```suggestion
The RAPIDS 23.08 release has made changes to end-user Docker images to improve functionality, performance, and maintainability.
```